script/map.js

```diff
@@ -0,0 +1,129 @@
+let map = L.map('map').setView([23.8, 120.9605], 7.5);
```

Comment:
@sakanana0210 
I didn't use this before.
Could you give me some example or reference materials related to this?
Thanks a lot!